### PR TITLE
add test for xyz writing

### PR DESCRIPTION
**Pull request by ltalirz**
testing that tags are handled as expected

(needed to make sure for aiida-lsmo)

### CODE REVIEW

**Review by danieleongari — DISMISSED** [review]

**Comment by ltalirz** [comment]
@yakutovicha feel free to merge

this is just adding a test and modularizing some functions

File: test/test_xyz.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+# Copyright (c), The AiiDA-CP2K authors.                                      #
+# SPDX-License-Identifier: MIT                                                #
+# AiiDA-CP2K is hosted on GitHub at https://github.com/cp2k/aiida-cp2k        #
```
Comment (yakutovicha) [inline test/test_xyz.py:5]:
```suggestion
# AiiDA-CP2K is hosted on GitHub at https://github.com/aiidateam/aiida-cp2k        #
```

**Review by yakutovicha — COMMENTED** [review]
looks good, just a small comment.

**Comment by ltalirz** [comment]
thanks!

**Pull request merged**